tracktolib/api.py

```diff
@@ -134,6 +134,12 @@ def _get_method_wrapper(cls: Endpoint, method: Method,
                         model: Type[B] | None = None,
                         openapi_extra: dict[str, Any] | None = None):
     def _set_method_wrapper(func: EnpointFn):
+        updated_openapi_extra = openapi_extra or {}
+        if hasattr(model, "__origin__"):
```

Comment:
You are trying to check if we are dealing with a list? Use get_origin instead (avoid id possible using __ attributes): https://docs.python.org/3/library/typing.html#typing.get_origin

Reply:
ok checking ignore last commit

Reply:
do not use type ignore also

Reply:
You can also use get_args https://docs.python.org/3/library/typing.html#typing.get_args